### ISSUE TITLE
Define `SYSTEM_UPDATE_DID`

### ISSUE THREAD

**Issue by FabioPinheiro** [input-output-hk/prism-did-method-spec#57]
Define `SYSTEM_UPDATE_DID`
This value should be hard code. It MUST NOT change and everyone should know it!

**Comment by yshyn-iohk** [comment]
@FabioPinheiro, could you please elaborate on the issue?
Is it a DID that should be used to migrate the data on the ledger to the higher version?

**Comment by FabioPinheiro** [comment]
Yes. I mean from my point of view this DID should be part of the specification.
No one can have a complete implementation of the protocol without that.
We need to define it and control the keys in a very secure place. I mean not even our team of devs should have access to those keys.

**Comment by mkbreuningIOHK** [comment]
@lohanspies @EzequielPostan to be involved to clarify customer value and implementation steps.

**Comment by FabioPinheiro** [comment]
This DID is a super user that is able to TAG in the blockchain that from this moment forward a new version of the protocol is to be followed. It informs all the PRISM nodes what is the version of the prism protocol.

Customer value - this is to be future proof in case ever we need to change the protocol.
Implementation steps - we just need to create one DID in the blockchain
